tests/bluetooth/crypto_toolbox/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(foo)
+
+target_sources(app PRIVATE
+  src/test_bt_crypto_toolbox.c
+)
+
+target_link_libraries(app PRIVATE
+  subsys__bluetooth__crypto_toolbox
```

Comment:
why is this needed ?

The app lib is inside the whole archive flag, as are Zephyr libraries such as `subsys__bluetooth__crypto_toolbox`.

Reply:
I think it fails to compile without this. What is the correct way to get this lib in?

Reply:
I was thinking the same but I tried to remove it and there is no problem.